ruby/Dockerfile

```diff
@@ -1,16 +1,11 @@
-# This is our DEVELOPMENT dockerfile.
-#
-# This uses Codelitt's Ruby 2.2 image found at:
-# https://github.com/codelittinc/dockerfiles/blob/master/ruby/Dockerfile
-FROM codelittinc/ruby:2.2
-MAINTAINER Codelitt, Inc.
```

Comment:
If you remove the FROM and MAINTAINER, then how does it know what the base image is?